### ISSUE TITLE
Dynamic page demotion e2e test failing

### ISSUE THREAD

**Issue by askervin** [intel/cri-resource-manager#659]
This is expected to pass:
```
cd test/e2e
./run_tests.sh policies.test-suite/topology-aware/c4pmem4/test04-dynamic-page-demotion-deprecated-syntax
```

`git bisect`ing suggests that after commit febcc6d99eb28cc39a60d4d2ecbc625721ddd725 the memory of the awk processes in the test is not moved to PMEM anymore.

For the record: this was bisected by using currently experimental [e2e patches](https://github.com/askervin/cri-resource-manager/tree/5Cq_e2e_k8scri) that allow installing cri-resmgr from `crirm_src` that is different from the source tree that contains the e2e tests to be executed.
```
experimental_crirm_test_e2e=~/go/src/github.com/askervin/cri-resource-manager/test/e2e
export crirm_src=~/go/src/github.com/intel/cri-resource-manager
export reinstall_cri_resmgr=1

cd $crirm_src
git bisect start
git bisect bad
git bisect good v0.4.0

# repeat until found:
git clean -dfx; go mod vendor; ( make DEBUG=1 && ( cd $experimental_crirm_test_e2e; ./run_tests.sh policies.test-suite/topology-aware/c4pmem4/test04-dynamic-page-demotion-deprecated-syntax ) && git bisect good ) || git bisect bad
```

**Comment by marquiz** [comment]
@askervin fixed?

**Comment by askervin** [comment]
Fixed in by PR #961 .
